apps/search/test/marionette/search_rocketbar_test.js

```diff
@@ -100,8 +110,18 @@ marionette('Search - Rocketbar Test', function() {
     assert.ok(!rocketbar.clear.displayed());
 
     // Perform a search
-    rocketbar.enterText('a test\uE006');
-    rocketbar.switchToBrowserFrame(server.url('sample.html'));
+    rocketbar.enterText(textToSearch, true);
+    rocketbar.switchToBrowserFrame(searchTemplate);
+  });
+
+  test('Browser search', function() {
+    home.waitForLaunch();
+    rocketbar.homescreenFocus();
+    var textToSearch = 'a test';
+    rocketbar.enterText(textToSearch, true);
```

Comment:
We could make the test closer to the user's intent if `rocketbar` had a method like:
```js
performSearchInBrowser(textToSearch):
    rocketbar.enterText(textToSearch, true);
    rocketbar.waitForBrowserFrame();
    rocketbar.appTitleFocus();
```